app/controller/settings.js

```diff
@@ -0,0 +1,23 @@
+var config = module.exports = {};
+
+var fs = require('fs');
+config.data = require('../config.json');
+
+config.update = function (req) {
+	// This is temporary
+	config.data.push({
+		id: config.data.length + 1,
+		name: 'new addition',
+		setting: req.body.setting
+	});
+
```

Comment:
What is this even doing we are not creating new settings? We need to update settings